### PR TITLE
Changes for v0.20 release

### PR DESCRIPTION
**Pull request by hoodmane**

### CODE REVIEW

File: docs/project/roadmap.md [hunk 1]
```diff
@@ -63,17 +63,6 @@ resolution.
 
 See issues {issue}`2045` and {issue}`1100`.
 
-## Switch to using wheels for Python packages
```
Comment (rth) [inline docs/project/roadmap.md:66]: I think we could keep these two, maybe just adjust the wording. 

Wheels work for us, but there is still an effort needed to get them accepted by PyPi, and there is likely standardization effort needed (including for backward compatibility policy)

For Rust, I see you are working on building cryptography, I imagine there are always things that could be improved size wise etc?

**Review by rth — APPROVED** [review]
Thanks!

File: docs/project/roadmap.md (hunk 1 above)
Comment (hoodmane) [reply docs/project/roadmap.md:66]: The size is actually fantastic: it's about 3x smaller than the corresponding package for native platform. And it really works: I assume Cryptography uses a wide range of features and their test suite passes thousands of checks up until chrome crashes.

Comment (hoodmane) [reply docs/project/roadmap.md:66]: So I could see modifying the wheels one to discuss some new issues. For Rust, we could update it to say address some specific issues. But if we find out issues with Rust, we could always put it back in. I figure it's good to update the roadmap frequently or else it doesn't mean much.

Comment (rth) [reply docs/project/roadmap.md:66]: Great results for Cryptography!

OK, up to you -- we can always edit this later if needed.

Comment (hoodmane) [reply docs/project/roadmap.md:66]: Yeah I'll just leave it out for now. Ideally I think we should focus a lot on the docs and examples for the next few months, hopefully as a part of that we can update the roadmap a bit.

**Pull request merged**